src/sql/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -197,6 +197,10 @@ export class SelectBox extends vsSelectBox {
 		return this._selectedOption;
 	}
 
+	public get label(): string {
+		return this._dialogOptions.find(s => s.value === this._selectedOption).text;
```

Comment:
I assume the `find` can never return undefined here?

Reply:
i'll add a test case to cover that potential too.